css/style.css

```diff
@@ -0,0 +1,193 @@
+    height: 100%;
+}
+
+.main-nav {
+    display: -webkit-box;
+    display: -ms-flexbox;
+    display: flex;
+}
+
+.mini-profile {
+    text-align: center;
+    border: solid #000 1px;
+    padding: 0 10px;
+}
+
+.mini-profile p {
+    line-height: 17px;
+    margin: 10px 5%;
+}
+
+.project-holder {
+    display: -webkit-box;
+    display: -ms-flexbox;
+    display: flex;
+    -ms-flex-wrap: wrap;
+    flex-wrap: wrap;
+    text-align: center;
+    list-style: none;
+}
+
+.project {
+    display: -webkit-box;
+    display: -ms-flexbox;
+    display: flex;
+    -webkit-box-pack: center;
+    -ms-flex-pack: center;
+    justify-content: center;
+    -webkit-box-flex: 1;
+    flex-shrink: 1;
+    -ms-flex-negative: 1;
+    flex-grow: 1;
+    -ms-flex-positive: 1;
+    flex-basis: 50%;
+    -ms-flex-preferred-size: 25%;
+    color: #fff;
+    background-image: url("https://placekitten.com/700/300");
+    background-size: cover;
+    min-height: 300px;
+    background-position: center;
+    position: relative;
+    padding: 0 15px;
+    border: solid black 1px;
+}
+
+.project:hover .project-over {
+    background: rgba(0,0,0,0.3);
+    height: 100%;
+    width: 100%;
+    color: rgba(212,175,55,1);
+    position: absolute;
+    top: 0;
+    left: 0;
+    padding: 0 15px;
+}
+
+.project1 {
+    background-image: url("https://images.pexels.com/photos/1393996/pexels-photo-1393996.jpeg?auto=compress&cs=tinysrgb&h=350");
+}
+
+.project2 {
+    background-image: url("https://images.unsplash.com/photo-1536499657529-2d09e5c71539?ixlib=rb-0.3.5&ixid=eyJhcHBfaWQiOjEyMDd9&s=ccc131c58f5da1e893e2605dda9de842&auto=format&fit=crop&w=500&q=60");
+}
+
+.project3 {
+    background-image: url("https://images.unsplash.com/photo-1536506252322-a4cb9cae8655?ixlib=rb-0.3.5&ixid=eyJhcHBfaWQiOjEyMDd9&s=79ac811fe55d39710f4d9d6cb2ac1d27&auto=format&fit=crop&w=500&q=60");
+}
+
+.project4 {
+    background-image: url("https://images.unsplash.com/photo-1536559692556-79e8be88e8ab?ixlib=rb-0.3.5&ixid=eyJhcHBfaWQiOjEyMDd9&s=97d54d56f7e8c363cda42b6651896f68&auto=format&fit=crop&w=500&q=60");
+}
+
+.about-me-holder {
+    display: block;
+    background: #eaeaea;
+    border: solid #000 1px;
+    padding: 5px 10px;
+    height: 100%;
+    line-height: 17px;
+}
+
+.about-me-holder:after {
+    width: 100%;
+    clear: both;
+    display: block;
+    content: ' ';
+}
+
+.about-me {
+    float: left;
+    width: calc(100% - 300px);
+}
+
+.contact-info {
+    float: right;
+    width: 270px;
+}
+
+@media only screen and (max-width: 600px) {
+    header {
+        display: -webkit-box;
+        display: -ms-flexbox;
+        display: flex;
+        -webkit-box-pack: center;
+        -ms-flex-pack: center;
+        justify-content: center;
+        background: #eaeaea;
+        padding: 4px;
+        border: solid #000 1px;
+    }
+    .project {
+        -ms-flex-negative: 1;
+        flex-shrink: 1;
+        -webkit-box-flex: 1;
+        -ms-flex-positive: 1;
+        flex-grow: 1;
```

Comment:
You have all these codes 
```-ms-flex-negative: 1;
        flex-shrink: 1;
        -webkit-box-flex: 1;
        -ms-flex-positive: 1;
        flex-grow: 1;
```
in your desktop version also so no need to indicate them again for smaller screen size.